lib/web_console/testing/erb_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'web_console/testing/fake_middleware'
+
+# This class is to pre-compile 'templates/*.erb'.
+class ERBHelper
+  def initialize(path)
+    @erb  = ERB.new(File.read path)
```

Comment:
You can see that the whole parentheses/no-parentheses situation is pretty inconsistent. That's why I would recommend you to put them everywhere (except the DSL-ish methods like require, etc.).